addons/website/views/snippets/snippets.xml

```diff
@@ -481,9 +481,11 @@
     </div>
 
     <!-- Mobile display options -->
-    <div data-option-name="showMobile" data-selector="section .row > div"
+    <div data-js="MobileVisibility" data-selector="section .row > div, section"
          data-exclude=".s_col_no_resize.row > div">
-        <we-checkbox string="Shown On Mobile" data-select-class="d-none d-md-block|" data-no-preview="true"/>
+        <we-button class="fa fa-fw fa-mobile o_we_link o_we_mobile"
```

Comment:
I am wondering... do you actually think this looks good? Or even understandable?

![Screenshot from 2021-02-19 18-11-55](https://user-images.githubusercontent.com/10338094/108539195-f6dec480-72df-11eb-8360-a0d83fb5109d.png)